Neos.Utility.MediaTypes/Tests/Unit/MediaTypesTest.php

```diff
@@ -173,7 +173,7 @@ public function mediaTypesWithAndWithoutParameters()
      * @test
      * @dataProvider mediaTypesWithAndWithoutParameters
      */
-    public function trimMediaTypeReturnsJustTheTypeAndSubTypeWithoutParameters($mediaType, $expectedResult)
+    public function trimMediaTypeReturnsJustTheTypeAndSubTypeWithoutParameters(string $mediaType, $expectedResult)
```

Comment:
Ah yes, the reason was that the dataProvider uses `null`, so that would have to be `string $expectedResult = null` for it to work then. Could be done though